src/mongo/db/storage/csv_file.cpp

```diff
@@ -29,10 +29,17 @@
 
 #include "mongo/db/storage/csv_file.h"
 
+#include <charconv>
+#include <fcntl.h>
+#include <filesystem>
 #include <fmt/format.h>
 #include <fstream>  // IWYU pragma: keep
 #include <memory>
 #include <string>
+#include <sys/mman.h>
+#include <sys/stat.h>
+#include <sys/types.h>
+#include <unistd.h>
```

Comment:
Why do we need sys/stats.h , sys/types.h, and unistd.h, and fcnti.h?
I get we need charcov for from_char(),
filesystem for filesystem, and sys/mman.h for mmap?

Reply:
As we discussed offline, stats.h & types.h are not necessary anymore. They were just leftovers from the previous iteration. unistd.h is necessary to use `open()` and fcntl.h is necessary to use `O_RDONLY`.